packages/devextreme/js/ui/chat.d.ts

```diff
@@ -194,6 +284,13 @@ export interface dxChatOptions extends WidgetOptions<dxChat> {
      * @public
      */
     items?: Array<Message>;
+    /**
+     * @docid
+     * @default { allowUpdating: false, allowDeleting: false }
```

Comment:
I assume we can omit this @default comment, the allowUpdating & allowDeleting have to contain default values inside. Seems, it will enough